plugins/catalog-react/src/components/EntityRefLink/humanize.ts

```diff
@@ -23,9 +23,13 @@ import {
 /** @public */
 export function humanizeEntityRef(
   entityRef: Entity | CompoundEntityRef,
-  opts?: { defaultKind?: string },
+  opts?: {
+    defaultKind?: string
+    skipDefaultNamespace?: boolean
```

Comment:
I feel that a `defaultNamespace` option would be a bit more natural here, defaulted to `DEFAULT_NAMESPACE` of course. It could be useful if you're creating a link from within the context of a different namespace too.

That just leaves the question of how to communicate that we never want to omit the namespace. I could for example imagine something like `defaultNamespace?: string | false` as one method? We could also have a constant, probably a symbol, that represents never skipping the namespace. We also don't usually use `null`, but this could be a situation where it makes sense. There's also the option of using the fact that JavaScript object differentiate between missing and undefined properties, so explicitly setting `defaultNamespace: undefined` could have that effect, seems a bit risky though.

@backstage/maintainers, any top pick?

```ts
humanizeEntityRef(ref, { skipDefaultNamespace: true });
humanizeEntityRef(ref, { defaultNamespace: false });
humanizeEntityRef(ref, { defaultNamespace:  ALWAYS_SHOW_NAMESPACE });
humanizeEntityRef(ref, { defaultNamespace: undefined });
humanizeEntityRef(ref, { defaultNamespace: null });
```